test/spec/SearchApi/Providers/GoogleReverseGeocoderSpec.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace spec\SearchApi\Providers;
+
+use SearchApi;
+use SearchApi\Models as Models;
+use SearchApi\Providers as Providers;
+use SearchApi\Support as Support;
+
+use PhpSpec\ObjectBehavior;
+use Prophecy\Argument;
+
+/**
+ * GoogleReverseGeocoderSpec - Spec integration test for the Google Reverse Geocoder (higher level functions)
+ */
+class GoogleReverseGeocoderSpec extends ObjectBehavior {
+  function it_is_initializable() {
+    $this->shouldHaveType( 'SearchApi\Providers\GoogleReverseGeocoder' );
+  }
+
+  function it_should_have_key_null() {
```

Comment:
Your testing the default case, which is why I would like to see the word 'default' in the test name. Otherwise it might sound more like you are just testing how the class works with a null api_key.